src/main/java/net/revelc/code/formatter/xml/XMLFormatter.java

```diff
@@ -39,13 +39,17 @@ public void init(Map<String, String> options, ConfigurationSource cfg) {
         String tabInsteadOfSpaces = options.get("tabInsteadOfSpaces");
         String tabWidth = options.get("tabWidth");
         String splitMultiAttrs = options.get("splitMultiAttrs");
+        String wellFormedValidation = options.get("wellFormedValidation");
 
         prefs.setMaxLineLength(maxLineLength != null ? Integer.valueOf(maxLineLength) : null);
         prefs.setWrapLongLines(wrapLongLines != null ? Boolean.valueOf(wrapLongLines) : null);
         prefs.setTabInsteadOfSpaces(tabInsteadOfSpaces != null ? Boolean.valueOf(tabInsteadOfSpaces) : null);
         prefs.setTabWidth(tabWidth != null ? Integer.valueOf(tabWidth) : null);
         prefs.setSplitMultiAttrs(splitMultiAttrs != null ? Boolean.valueOf(splitMultiAttrs) : null);
 
+        if (wellFormedValidation != null)
```

Comment:
@jam01 Can you add curry brackets around the if statement here?  Then we are good to merge :)

Reply:
done :)

Reply:
Thanks, file needs a format but I'll take this and format afterwards.  Appreciate it.